### DISCUSSION TITLE
Ephemeral messages in v2.0

### DISCUSSION THREAD

**Discussion by moon-dew** [Rapptz/discord.py#7050] [Ideas] [answered]
I didn't see this on the project list, so I was wondering whether ephemeral messages will be supported in v2.0? And if that's not planned, I think adding it would be very nice

**Comment by pikaninja** [comment, marked as answer]
ephemeral messages can only be used in slash commands or interactions such as buttons. Buttons *have* been added with  ephemeral message support(such as interaction.response.send_message(content = "hi",  ephemeral = True)), as for slash commands that is on the project list.

**Reply by moon-dew** [reply]
I see, great!
